glossary/README.md

```diff
@@ -0,0 +1,3 @@
+# Cosmos Blockchain Glossary
```

Comment:
```suggestion
<!--
  Title: Cosmos Blockchain Glossary
  Description: A curated list of blockchain terms, words, and language that are used in the Cosmos universe. 
  -->
# Cosmos Blockchain Glossary
```

Reply:
is this metadata useful at all for SEO? if not, ignore. 
In regular markdown, we use metadata in the docs to help SEO

Reply:
We shouldn't include any YAML front matter in this readme file.